lms/urls.py

```diff
@@ -1049,3 +1049,8 @@
 urlpatterns += [
     path('api/notifications/', include('openedx.core.djangoapps.notifications.urls')),
 ]
+
+# Content tagging
+urlpatterns += [
+    path('api/content_tagging/', include(('openedx.features.content_tagging.urls'))),
```

Comment:
Do we need the taxonomy API in the LMS?

Reply:
I'm not sure. Can you confirm @pomegranited?

Reply:
@bradenmacdonald No, we don't need the taxonomy API in the LMS.

But note that if we remove these endpoints from the LMS:

* The content_tagging APIs disappear from the swagger-generated api-docs
* The views tests need to be wrapped in [@skip_unless_cms](https://github.com/openedx/edx-platform/blob/0935e08a696204e7883f1864e7fc0f0488a3c878/openedx/core/djangolib/testing/utils.py#L244), and running them manually in the devstack requires some extra steps.

I'm fine either way.

Reply:
I don't think we should allow people to make edits to content using the LMS API, and I do slightly worry that if we expose the endpoints in both, some people will use them from the LMS. But it doesn't seem like a big deal so I'm also fine either way. Having the swagger docs is a nice thing. Do the swagger docs show CMS URLs though, if you access them from the LMS? 

> running them manually in the devstack requires some extra steps.

Oh really? I haven't used devstack in a while. For tutor you always pick to run tests in LMS or CMS so there's no extra step either way.

Reply:
> Do the swagger docs show CMS URLs though, if you access them from the LMS?

I don't think so. Is another port/servervice.